Basic/APIDiorama/Assets/Scripts/ClientNetworkAnimator.cs

```diff
@@ -0,0 +1,9 @@
+using Unity.Netcode.Components;
```

Comment:
Now that it is part of the Utilities package, this can be replaced by a reference to it

Reply:
yes, you can find it [here](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/tree/main/Packages/com.unity.multiplayer.samples.coop/Utilities/Net/ClientAuthority). To use it, you just need to add "com.unity.multiplayer.samples.coop": "https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop", to the project's package manifest file

Reply:
Done!